design-toolkit/components/src/components/color-picker/index.tsx

```diff
@@ -156,7 +148,9 @@ function ColorSwatchPicker({
   const colorMap = useMemo(() => new Map(), []);
   return (
     <AriaListBox
-      className={cn('flex flex-wrap gap-s', className)}
+      className={composeRenderProps(className, (className) =>
+        swatchPicker({ className }),
+      )}
       style={props.style}
```

Comment:
Can you destructure these props please, and move the default value up to where the destructuting occurs?

Reply:
Done.